velox/type/UnscaledLongDecimal.h

```diff
@@ -161,6 +161,22 @@ struct UnscaledLongDecimal {
     return *this;
   }
 
+  static inline void serialize(
+      char* serializedData,
+      const UnscaledLongDecimal& longDecimal) {
+    *reinterpret_cast<uint64_t*>(serializedData) =
```

Comment:
What if we just do this?
```
*reinterpret_cast<int128_t*>(serializedData) = longDecimal.unscaledValue();
```

Reply:
I will give it a try.

Reply:
I rebased the minmax agg PR on top of this PR. The linux tests hit segfault https://app.circleci.com/jobs/github/facebookincubator/velox/127040 .
Looks like reinterpret_cast<int128*> is not helping, after reverting to copying parts fixed the issue.